presto-main/src/main/java/com/facebook/presto/operator/aggregation/LongSumAggregation.java

```diff
@@ -31,7 +34,12 @@ private LongSumAggregation() {}
     public static void sum(NullableLongState state, @SqlType(StandardTypes.BIGINT) long value)
     {
         state.setNull(false);
-        state.setLong(state.getLong() + value);
+        try {
+            state.setLong(Math.addExact(state.getLong(), value));
```

Comment:
Add a helper "addExact" that does the Math call and translates the exception rather than copying this everywhere